src/ui/screens/getHelp/developerMenu/ViewTelemetryDebugging.qml

```diff
@@ -72,6 +72,20 @@ MZViewBase {
             }
         }
 
+        MZCheckBoxRow {
+            id: checkBoxRowShortSessionTimer
+
+            Layout.fillWidth: true
+            Layout.rightMargin: MZTheme.theme.windowMargin
+            labelText: "3 minute timers for session pings"
```

Comment:
Please not here that this will apply to both the vpnsession and daemonsession ping.

Reply:
Good call, thank you!